generators/server/templates/src/main/java/package/config/lazy/LazyInitBeanFactoryPostProcessor.java.ejs

```diff
@@ -0,0 +1,73 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package  <%=packageName%>.config.lazy;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+/**
+ * Lazy initialize all Spring Beans, which boosts up start-up performance.
+ *
+ * This class originally comes from https://github.com/dsyer/spring-boot-allocations where it is written
+ * it can be copied on a project.
+ *
+ * As this is an experimental feature, we only recommend to use it in "dev" mode for the moment.
+ */
+@Component
+@Profile("dev")
```

Comment:
Use constant : JHipsterConstants.SPRING_PROFILE_DEVELOPMENT instead of "dev"

Reply:
done